spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,9 +323,43 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response!(app_id: nil, platform: nil, review_id: nil, response: "")
```

Comment:
i know something like 

`def def create_developer_response!(app_id:, platform: , review_id: , response: "") `

but it does not work on some older ruby's - i did a similar mistake on the IAP-Implementation, where we reverted to default values.